CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,337 @@
+  As community facilitators, we also strive to apply this value in creating 
+  Communities of Practice that welcomes and support members of all backgrounds 
+  and identities. This includes, but is not limited to
+  members of any race, ethnicity, culture, national origin, colour, immigration
+  status, social and economic class, educational level, sex, sexual orientation,
+  gender identity and expression, age, size, family status, political belief,
+  religion, and mental and physical ability.
+
+- 4.3 **Be considerate**: Your work will be used by other people, and you in turn
+  will depend on the work of others. Any decision you take will affect users and
+  colleagues, and you should take those consequences into account when making
+  decisions. Remember that we're an open source project. This means that contributors
+  might not be communicating in their primary language.
+
+- 4.4 **Be respectful**: Not all of us will agree all the time, but disagreement is
+  no excuse for poor behaviour and poor manners. We might all experience some
+  frustration now and then, but we cannot allow that frustration to turn into a
+  personal attack. It’s important to remember that a project where people feel
+  uncomfortable or threatened is not a productive one.
+
+- 4.5 **Be careful in the words that we choose**: We are
+  professionals, and we conduct ourselves professionally. Be kind to others. Do
+  not insult or put down other participants. Harassment and other exclusionary
+  behaviour aren't acceptable. This includes, but is not limited to Violent
+  threats or language directed against another person, Discriminatory jokes and
+  language, Posting sexually explicit or violent material, Posting (or
+  threatening to post) other people’s personally identifying information
+  (“doxing”), Personal insults, especially those using racist or sexist terms,
+  Unwelcome sexual attention, Advocating for or encouraging, any of the above
+  behaviour, and Repeated harassment of others. In general, if someone asks you to
+  stop, then stop.
+
+- 4.6 **Try to understand why we disagree**: Disagreements, both social and
+  technical, happen all the time. We must resolve disagreements
+  and differing views constructively. Remember that we’re different. Diversity
+  contributes to the strength of our work, which is composed of people from
+  a wide range of backgrounds. Different people have different perspectives on
+  issues. Being unable to understand why someone holds a viewpoint doesn’t mean
+  that they’re wrong. Don’t forget that it is human to err and blaming each
+  other doesn’t get us anywhere. Instead, focus on helping to resolve issues and
+  learning from mistakes.
+
+## 5. Examples of expected and unacceptable behaviour
+
+**5.1** Examples of behaviour that contributes to a positive environment for our
+community include:
+
+1. Demonstrating empathy and kindness toward other people
```

Comment:
The numbers are displayed as a series of 1's, not sure why on my end.

Reply:
This is a markdown "hack". If you look at the rendered copy, you'll see that the numbers progress as you'd expect. This means that authors don't have to keep track of how many bullets they have, and can also help mitigate issues where instructions might say "refer to step 3", but then a new PR adds a new step before so step 3 isn't step 3 anymore, etc.

Reply:
Thanks Sarah for addressing this!

Reply:
Thanks so much @sgibson91 for the clarification! 👍